Shared/Samples/Generate offline map with local basemap/README.metadata.json

```
@@ -0,0 +1,31 @@
+{
+    "category": "Edit and Manage Data",
+    "description": "Take a web map offline, but instead of downloading an online basemap, use one which is already on the device.",
```

```suggestion
    "description": "Use the `OfflineMapTask` to take a web map offline, but instead of downloading an online basemap, use one which is already on the device.",
```
metadata's description is the same as the README's first section.